docs/en/observability/synthetics-private-location.asciidoc

```diff
@@ -0,0 +1,130 @@
+[[synthetics-private-location]]
+= Add a private location
+
+IMPORTANT: This is only relevant to monitors created using the {uptime-app} or project monitors.
+
+beta[]
+
+Private locations allow you to run monitors from your own premises.
+	
+Before you create a private location, you need to set up {fleet}, and connect an agent to it.
+Then, you can create a private location in the {uptime-app} UI.
+Now, whenever you access the monitor creation page, this private location will appear in the list
+of locations to which you can deploy monitors.
+
+TIP: You can run the same monitor both from Elastic's global managed infrastructure _and_ your private locations to compare results.
+
+[discrete]
+[[synthetics-private-location-fleet-agent]]
+== Set up {fleet-server} and {agent}
+
+To deploy monitors to private locations, you must complete two prerequisites:
+
+* *Set up {fleet-server}*: If you are using {ecloud}, {fleet-server} will already be provided and you can skip this step.
+To learn more, see <<set-up-fleet,{fleet}>>.
+* **Create an agent policy**:  For more information on agent policies and creating them,
```

Comment:
Do we need to say you need to apply the policy to a live agent instead of just saying they need to create a policy?